composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "curl/curl": "^2.5",
```

Comment:
why do you need curl?

Reply:
I need it for using api to validate and verify email id.

Used it here: https://github.com/faizan-nasir/FT2-77-PHP-Basics/pull/1/files#r1508935278